resotocore/resotocore/web/api.py

```diff
@@ -957,6 +961,21 @@ async def graph_query_model_from_request(self, request: Request) -> Tuple[GraphD
         section = section_of(request)
         query_string = await request.text()
         graph_name = GraphName(request.match_info.get("graph_id", "resoto"))
+        raw_at = request.query.get("at")
+        try:
+            at = date_parser.parse(raw_at) if raw_at else None
```

Comment:
If the user enters a wrong date, he will read from the current snapshot.
I would say the user should get an error in this case.

Reply:
Also: consider allowing relative timestamps.